tests/dynamic_data/publications/layer_wfst/new_attribute_test.py

```diff
@@ -192,7 +191,9 @@ class TestNewAttribute(base_test.TestSingleRestPublication):
             publication=lambda cls, parametrization: Publication(
                 workspace=cls.workspace,
                 type=cls.publication_type,
-                name=f"lr_{'_'.join(v.publ_name_part for v in parametrization.values_list)}"),
+                name=f"lr_{'_'.join(v.publ_name_part for v in parametrization.values_list)}",
+                uuid='0367efa3-0992-4035-bb23-628475edde14',
```

Comment:
Please mention this test fails with some UUIDs, refer to refer to tests/dynamic_data/publications/issues/gs_wfst_update_replace.py